include/PreCharge/dev/MAX22530.hpp

```diff
@@ -0,0 +1,34 @@
+#include <EVT/io/SPI.hpp>
+#include <PreCharge/PreCharge.hpp>
+#include <cstdint>
+
+namespace PreCharge {
+
+/**
+ * Handles reading the voltage values & converting it to decivolts from the
+ * MAX22530.
+ */
+class MAX22530 {
+public:
+    /**
+     * Creates a new MAX22530 which will read a raw ADC voltage and convert it
+     * to decivolts.
```

Comment:
NIT: There should be a newline between the summary and parameters in each method comment